src/home/NotificationCenter.tsx

```diff
@@ -0,0 +1,118 @@
+import { LayoutChangeEvent, StyleSheet, Text, View } from 'react-native'
+import Animated, { useAnimatedScrollHandler, useSharedValue } from 'react-native-reanimated'
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+import type { Notification } from 'src/home/NotificationBox'
+import { useNotifications } from 'src/home/NotificationBox'
+import ThumbsUpIllustration from 'src/icons/ThumbsUpIllustration'
+import { Screens } from 'src/navigator/Screens'
+import useScrollAwareHeader from 'src/navigator/ScrollAwareHeader'
+import { StackParamList } from 'src/navigator/types'
+import fontStyles from 'src/styles/fonts'
+import { Spacing } from 'src/styles/styles'
+
+type NotificationsProps = NativeStackScreenProps<StackParamList, Screens.NotificationCenter>
+
+const renderItem = ({ item }: { item: Notification }) => {
+  return (
+    <View testID={`NotificationView/${item.id}`} key={item.id} style={styles.listItem}>
+      {item.element}
+    </View>
+  )
+}
+
+const keyExtractor = (item: Notification) => item.id
+
+const renderItemSeparator = () => <View style={styles.itemSeparator} />
+
+export default function Notifications({ navigation }: NotificationsProps) {
+  const safeAreaInsets = useSafeAreaInsets()
+
+  const { t } = useTranslation()
+
+  const title = t('notifications')
+
+  const [headerYPosition, setHeaderYPosition] = useState(0)
+  const [headerHeight, setHeaderHeight] = useState(0)
+
+  const handleMeasureHeaderHeight = (event: LayoutChangeEvent) => {
+    setHeaderYPosition(event.nativeEvent.layout.y)
+    setHeaderHeight(event.nativeEvent.layout.height)
+  }
+
+  const scrollPosition = useSharedValue(0)
+
+  useScrollAwareHeader({
+    navigation,
+    title,
+    scrollPosition,
+    startFadeInPosition: headerYPosition,
+    animationDistance: headerHeight,
+  })
+
+  const handleScroll = useAnimatedScrollHandler((event) => {
+    scrollPosition.value = event.contentOffset.y
+  })
+
+  const renderHeader = () => (
+    <Text style={styles.title} onLayout={handleMeasureHeaderHeight}>
+      {title}
+    </Text>
+  )
+
+  const renderEmptyState = () => (
+    <View testID="NotificationCenter/EmptyState" style={styles.emptyStateContainer}>
+      <ThumbsUpIllustration />
+      <Text style={styles.emptyStateLabel}>{t('youAreAllCaughtUp')}</Text>
+    </View>
+  )
+
+  const contentContainerStyle = { paddingBottom: safeAreaInsets.bottom + Spacing.Regular16 }
+
+  const notifications = useNotifications()
+
+  return (
+    <Animated.FlatList
+      style={styles.container}
+      ListHeaderComponent={renderHeader}
+      ListEmptyComponent={renderEmptyState}
+      ItemSeparatorComponent={renderItemSeparator}
+      data={notifications}
+      renderItem={renderItem}
+      keyExtractor={keyExtractor}
+      onScroll={handleScroll}
+      contentContainerStyle={contentContainerStyle}
+    />
+  )
+}
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+  },
+  title: {
+    ...fontStyles.h1,
```

Comment:
nit: maybe this is not needed?

Reply:
yeah, `h1` is almost overwritten, the only property not being replaced is `color` 😅 

i've replaced h1 spreading with an evident `color` property